sunday.tex

```diff
@@ -49,9 +49,9 @@ \section*{Sunday Sessions Descriptions}\label{sunday-descriptions}
   "`Mapathon, mapathon, mapathon!"'%
 }{%
 }{%
-  Who benefits from the mapathons: between (over)communication and (over)attribution, critical
+  Who benefits from mapathons?: between (over)communication and (over)attribution, critical
```

Comment:
Shouldn't "between" be written with a capital B and the colon removed?

Reply:
Indeed. Have updated accordingly.

Reply:
Well I thought I had....cannot see it having coming through to the pr yet

Reply:
I will go through the whole booklet with an regular expression to look for lowercase after colon and check if a sentence ends at the colon.